android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -80,8 +79,8 @@ private void downloadFeed() {
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(feed -> {
-                    feedItems.clear();
-                    feedItems.addAll(feed.getData());
+                    feedFragmentViewModel.getFeedList().getValue().clear();
```

Comment:
You shouldn't mutate the ViewModel state, the UI should replicate the data

Reply:
Ok I will make methods in ViewModel to do this and call it here?